### ISSUE TITLE
meteorBabelHelpers is Undefined in the Cordova Bundle (iOS) in releases > 1.8.1

### ISSUE THREAD

**Issue by d0liver** [meteor/meteor#11123]
I'm attempting to build for iOS (with Cordova on macOS) on any Meteor release above 1.8.1. Running the app in the iOS simulator produces a `meteorBabelHelpers is undefined` error coming from multiple Meteor packages using coffeescript like https://github.com/softwarerero/meteor-accounts-t9n and https://github.com/arillo/meteor-flow-router-helpers. I did some digging and it looks like `meteorBabelHelpers` was supposed to be exposed to node_modules in the legacy bundle here: https://github.com/meteor/meteor/pull/10596.

I'd guess that what's happening is that these are old packages that are now compiling with coffeescript@2.x and leaking out newer syntax to Babel (since coffeescript 2.x doesn't attempt to polyfill newer syntaxes), but `meteorBabelHelpers` isn't available in the legacy environment for Meteor packages.

**Comment by hexsprite** [comment]
Hey, I had this same problem. I ended up forking the packages and adding this at the top of each .coffee file:

```
meteorBabelHelpers = Package.modules.meteorBabelHelpers
```

which solved the issue for me. I didn't have that many coffeescript packages.

I had filed some issues about this but there wasn't much interest in fixing it ;)

**Comment by d0liver** [comment]
@hexsprite Thanks very much. I was able to get this working doing what you suggested. I'm going to leave this open though, because I think that this effectively breaks compatibility with CoffeeScript for users unless they're willing to maintain their own forks of integral packages.

**Comment by filipenevola** [comment]
Is the workaround suggested by @hexsprite enough?

**Comment by xet7** [comment]
@filipenevola 

No, [legacy bundle and browser detection code is broken](https://github.com/meteor/meteor/issues/11059#issuecomment-634029300) so that's why Meteor/Wekan does not currently work in IE11 at all. Those dependencies should be updated to upstream, or alternative working dependencies proposed.

**Comment by filipenevola** [comment]
Sorry @xet7, I'm confused here, are they the same issue?

Our devel branch (1.11.1) is passing on IE11 in our regressions tests running on BrowserStack as you can see here https://app.circleci.com/pipelines/github/meteor/meteor/450/workflows/bbcf6bcc-ccac-4569-8206-527311c70f20/jobs/49616

<img width="624" alt="Test Group 11 (49616) - meteor:meteor 2020-11-12 16-11-10" src="https://user-images.githubusercontent.com/437372/98991076-b5c95e00-2501-11eb-9dcb-78f65da07a70.png">

Is our BrowserStack test reporting the incorrect result? Or is it something else like in this case https://github.com/meteor/meteor/issues/11163?

**Comment by xet7** [comment]
@filipenevola 

After I updated Wekan to meteor devel at https://github.com/wekan/wekan/commit/9087bd657385b9a94ff8e9ce695ba41b76199f6b , I still see undefined errors on IE at https://friend.wekan.team (test server running meteor devel, you can register account) like this:

![meteor-undefined](https://user-images.githubusercontent.com/15545/99017853-75013180-2561-11eb-9210-073f7f5c9006.png)

Those errors are not visible at Firefox and MS Legacy Edge.

Maybe it's some dependency?

**Comment by StorytellerCZ** [comment]
Just to confirm, this only issue in IE11?

**Comment by nachocodoner** [comment]
Meteor 3 no longer compiles `web.browser.legacy` by default, and modern Cordova bundles natively support these modern ES targets. We are closing this as obsolete.
